x-pack/filebeat/input/awss3/state.go

```diff
@@ -0,0 +1,63 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package awss3
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// State is used to communicate the reading state of a file
+type State struct {
+	Id           string    `json:"id" struct:"id"`
+	Bucket       string    `json:"bucket" struct:"bucket"`
+	Key          string    `json:"key" struct:"key"`
+	LastModified time.Time `json:"last_modified" struct:"last_modifed"`
+	Size         int64     `json:"size" struct:"size"`
+	Stored       bool      `json:"stored" struct:"stored"`
```

Comment:
Please add a godoc comment about what "Stored" indicates. I think it means that the object's contents have been published and ACKed by the output.

Reply:
👍